### ISSUE TITLE
Deleting data from boxes using removeMany is increasing the DB size.

### ISSUE THREAD

**Issue by raphire08** [objectbox/objectbox-dart#506]
### Basic info (please complete the following information):

- ObjectBox version: [e.g. 1.6.2]
- Flutter/Dart SDK: [e.g. 2.0.0, or the output of `dart --version` or `flutter --version`]

```
Flutter 3.3.7 • channel stable • https://github.com/flutter/flutter.git
Framework • revision e99c9c7cd9 (3 months ago) • 2022-11-01 16:59:00 -0700
Engine • revision 857bd6b74c
Tools • Dart 2.18.4 • DevTools 2.15.0
```

- Null-safety enabled: yes
- Reproducibility: always
- OS: iOS
- Device/Emulator: Iphone XR

### Steps to reproduce

1. Enter any data in couple of boxes
2. Delete the data in transaction using removeMany method

### Expected behavior

Total size of data.mdb should decrease but it is increasing

### Code

If applicable, add code to help explain your problem.

```
 _db.store.runInTransaction(TxMode.write, () {
      _db.store.box<Box1>().removeMany(ids1);
      _db.store.box<Box2>().removeMany(ids2);
});
```

**Comment by greenrobot-team** [comment]
This is by design. I invite you to read https://docs.objectbox.io/faq#how-does-objectbox-use-disk-space-and-can-i-reclaim-disk-space for an explanation and a solution.

TL;DR: re-using existing file areas is much more efficient than shrinking and growing a file; used file storage is likely to be used again in the future; if absolutely required clean up, close the store, delete the database files and start over.
